sorc/build_workflow_utils.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -eux
 
+source ./machine-setup.sh > /dev/null 2>&1
```

Comment:
This line is sourced on 14.  I presume you are looking to set the variable `target` here.  You don't need since the calling script `build_all.sh` sets this variable and passes it down.  See: https://github.com/lgannoaa/global-workflow/blob/b19ccea00d59a4c8ea13d9f8ffb00fbe7bfc9d49/sorc/build_all.sh#L154

Reply:
Modification made as requested